src/main/java/seedu/recipe/logic/commands/ImportCommand.java

```diff
@@ -51,13 +53,19 @@ public CommandResult execute(Model model) throws CommandException {
             }
             // Validate uniqueness, add to model
             ObservableList<Recipe> currentRecipes = model.getFilteredRecipeList();
-            if (importedRecipeList.stream().anyMatch(currentRecipes::contains)) {
-                throw new CommandException(DUPLICATE_VALUES);
+            for (Recipe recipe : importedRecipeList) {
+                if (currentRecipes.stream().noneMatch(recipe::isSameRecipe)) {
+                    model.addRecipe(recipe);
+                } else {
+                    hasDuplicate = true;
+                }
             }
-            importedRecipeList.forEach(model::addRecipe);
         } catch (IllegalValueException e) {
             throw new CommandException(e.getMessage());
         }
+        if (isDuplicate()) {
+            return new CommandResult(DUPLICATE_VALUES);
```

Comment:
import function works as expected and correctly imports the json file while ignoring duplicate recipes. however, the command message is not correctly formatted (it doesn't include the json file name):
![image](https://user-images.githubusercontent.com/8911178/230540277-4932b310-f490-453f-a394-c4ca3f014053.png)

Reply:
Resolved to remove file name (since user imports the file)